src/me/ryanhamshire/PopulationDensity/DataStore.java

```diff
@@ -869,7 +869,10 @@ private void loadMessages()
         this.addDefault(defaults, Messages.RegionNamesOnlyLettersAndNumbers, "Region names may not include symbols or punctuation.", null);
         this.addDefault(defaults, Messages.RegionNameConflict, "There's already a region by that name.", null);
         this.addDefault(defaults, Messages.NoMoreRegions, "Sorry, you're in the only region.  Over time, more regions will open.", null);
+	this.addDefault(defaults, Messages.InviteAlreadySent, "{0} may now use /visit {1} to teleport to your home post.", "0: invitee's name, 1: inviter's name");
```

Comment:
whitespace :(

If I ever decide to overhaul this plugin, I'll be sure to fix this lol.